applications/asset_tracker/src/main.c

```diff
@@ -528,8 +537,10 @@ static void agps_request_convert(
 	struct nrf_modem_gnss_agps_data_frame *dest,
 	const struct gps_agps_request *src)
 {
-	dest->sv_mask_ephe = src->sv_mask_ephe;
-	dest->sv_mask_alm = src->sv_mask_alm;
+	/* It is not necessary to request ephemerides and almanacs when P-GPS is enabled. */
+	dest->sv_mask_ephe = IS_ENABLED(CONFIG_NRF_CLOUD_PGPS) ? 0 : src->sv_mask_ephe;
+	dest->sv_mask_alm = IS_ENABLED(CONFIG_NRF_CLOUD_PGPS) ? 0 : src->sv_mask_alm;
```

Comment:
That's one way to do it, and I'd approve it.  But this application has not been converted to using the GNSS modem interface directly, at which point, this code will need to be redone.

Reply:
I don't think ATv1 will be converted to the new API because it's already marked as deprecated.